rhel7/guide.xslt

```diff
@@ -28,6 +28,7 @@
       <xsl:apply-templates select="document('profiles/stig-rhel7-disa.xml')" />
       <xsl:apply-templates select="document('profiles/stig-rhvh-upstream.xml')" />
       <xsl:apply-templates select="document('profiles/ospp-rhel7.xml')" />
+      <xsl:apply-templates select="document('profiles/ospp42-draft-rhel7.xml')" />
```

Comment:
couple things I don't like about the name. I wish it didn't contain rhel7 in it. That is already implied by bechmark file name - ssg-rhel7-ds.xml, why specify it again?

"draft" is good because it means people shouldnt' rely on it. I wish the version name wasn't there.

I guess what I am trying to get at is that in the future let's please rename profiles to small descriptive names that are stable across releases. And avoid all duplicates.

Reply:
@mpreisler how would you suggest having version names that the profiles correspond to? ospp42 is clearly different than ospp.

Reply:
ok nvm about the version then

repeating the product name in profile IDs is not useful though and we do it everywhere... It's maybe useful for improving typing speed of our users :-)